### ISSUE TITLE
Map context menu only works once

### ISSUE THREAD

**Issue by gitToad** [dceejay/RedMap#259]
Per example in the docs, I inited a dashboard ui-worldmap with
```
msg.payload = {
        command: {
            ....
            hiderightclick: false,
            contextmenu: '<button name = "Clicker" onclick =\'feedback(this.name,"ping!",null,true)\'>Click me</button>'
        }
    }
```

works great the first time, ie I have a "worldmap in" node that receives the action=feedback message fine.
For reference, console shows **FB3"Clicker""ping!"null true** after first click on contextmenu button 

Even if I do nothing with that message (ie no changes to map), the next time I do a right click for context menu, the context menu content is corrupted (shows seemingly random characters: "ppCplpipcpkp pmpep" etc.)

It is as if calling the feedback function from the map rather than a marker messes up the context menu.

Context menus on markers work fine. This is re context menus on the map only

NodeRed 3.1.3
node-red-contrib-web-worldmap 4.6.1
Mac OS 14.3
Safari 17.3 (same result in latest Chrome and Firefox for Mac)

Not sure if relevant, but just in case: I did notice several 404 errors on load of the dashboard page:
```
[Info] milsymbol.js 2.0.0 - Copyright (c) 2018 Måns Beckman  http://www.spatialillusions.com (milsymbol.js, line 27)
[Log] CONNECT TO – "/worldmap/socket" (worldmap.js, line 116)
[Log] IN an iframe (worldmap.js, line 354)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (sockjs.min.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (togeojson.umd.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (leaflet-rotate.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (VectorTileLayer.umd.min.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (esri-leaflet.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (leaflet.antimeridian-src.js.map, line 0)
[Log] CONNECTED (worldmap.js, line 92)
```

**Comment by gitToad** [comment]
Actually I noticed the contextmenu content is NOT random. The seemingly gibberish string "**ppCplpipcpkp pmpep**" actually seems to contain the text "**Click me**" with a bunch of p's interspersed. Could this be a case of some dual byte Unicode to ASCII mistranslation ?

**Comment by dceejay** [comment]
Fixed in 4.6.2
